packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -80,7 +80,16 @@ export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFor
 
     return (
     <View style={styles.container}>
-      <Text style={styles.title}>{text.formTitle}</Text>
+      <View style={styles.titleContainer}>
+          <Text style={styles.title}>{text.formTitle}</Text>
+          {config.showBranding && (
+            <Image
```

Comment:
Can't we use an svg image?

from here: https://sentry.io/branding/

we could have an svg image, like this one:
```
<svg class="css-lfbo6j e1igk8x04" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 44" width="400" height="352" style=""><path d="M29,2.26a4.67,4.67,0,0,0-8,0L14.42,13.53A32.21,32.21,0,0,1,32.17,40.19H27.55A27.68,27.68,0,0,0,12.09,17.47L6,28a15.92,15.92,0,0,1,9.23,12.17H4.62A.76.76,0,0,1,4,39.06l2.94-5a10.74,10.74,0,0,0-3.36-1.9l-2.91,5a4.54,4.54,0,0,0,1.69,6.24A4.66,4.66,0,0,0,4.62,44H19.15a19.4,19.4,0,0,0-8-17.31l2.31-4A23.87,23.87,0,0,1,23.76,44H36.07a35.88,35.88,0,0,0-16.41-31.8l4.67-8a.77.77,0,0,1,1.05-.27c.53.29,20.29,34.77,20.66,35.17a.76.76,0,0,1-.68,1.13H40.6q.09,1.91,0,3.81h4.78A4.59,4.59,0,0,0,50,39.43a4.49,4.49,0,0,0-.62-2.28Z" fill="#362d59"></path></svg>
```

I like the idea of using the svg icon because it's easy to customize the logo color, so if a user decides to use a dark theme, the Sentry logo will not be hidden (by that we would need to allow them to change the sentry color or maybe some logic to know if the background color is dark or bright)

Reply:
That's a good point. I defaulted to PNG due to [the discussion with @krystofwoldrich on the form PR](https://github.com/getsentry/sentry-react-native/pull/4328#issuecomment-2521057407).
I think that using an svg directly might require a library which I'm not sure we want to import in core 🤔 
Maybe I can try using a JSX with a convertor like https://www.svgviewer.dev/svg-to-react-jsx. Wdyt?

Reply:
We can't use SVG, because there is no out of the box support and we can't add dependency just to render our logo and vendoring a library would also not work due to size and potential build issues.

https://www.svgviewer.dev/svg-to-react-jsx would work only for web